lib/earmark_parser/helpers/lookahead_helpers.ex

```diff
@@ -26,6 +29,7 @@ defmodule EarmarkParser.Helpers.LookaheadHelpers do
   opening backtix
   """
   # (#{},{_,_}) -> {_,_}
+  @spec still_inline_code(EarmarkParser.Line.t(), {backtix, non_neg_integer()}) :: {backtix | nil, non_neg_integer()}
```

Comment:
so here is my idea about typing this

```elixir
@type backtix_info_t :: {maybe(String.t()), non_neg_integer()}

```
or
```elixir
@type backtix_info_t :: {String.t()), non_neg_integer()}
@type backtix_info_t? :: {maybe(String.t()), non_neg_integer()}
```
and maybe, as I do now in my other projects

`lib/earmark_parser/types.ex`

```elixir
    quote do
        @type maybe(t) ::  t | nil
        ...
    end
```